m3u_parser/m3u_parser.py

```diff
@@ -452,18 +443,17 @@ def _get_m3u_content(self) -> str:
             line = "#EXTINF:-1"
             if "tvg" in stream_info:
                 for key, value in stream_info["tvg"].items():
-                    if value:
-                        line += ' tvg-{}="{}"'.format(key, stream_info["tvg"][key])
-            if stream_info.get("logo"):
+                    line += ' tvg-{}="{}"'.format(key, value)
+            if "logo" in stream_info:
                 line += ' tvg-logo="{}"'.format(stream_info["logo"])
-            if stream_info.get("country", {}).get("code"):
+            if "country" in stream_info:
                 line += ' tvg-country="{}"'.format(stream_info["country"]["code"])
-            if stream_info.get("language", {}).get("name"):
-                line += ' tvg-language="{}"'.format(stream_info["language"]["name"])
-            if stream_info.get("category"):
+            if "language" in stream_info:
+                line += ' tvg-language="{}"'.format(stream_info["language"]["code"])
```

Comment:
Here I had kept the name and you replaced it with the code again ?? Any reason to do so ?? I followed the m3u files present here https://github.com/iptv-org/iptv/tree/master/channels.

Reply:
It was a mistake, sorry @pawanpaudel93. It should be fixed now (f34cd0d7221260bbbda52b655b531de95eddcbab).